### PR TITLE
Adding new PNI nav

### PR DESCRIPTION
**Pull request by gvn**
related to #497

### CODE REVIEW

**Comment by gvn** [comment]
@beccaklam Can you give this a design review? 

https://advocacy-mozilla-org-st-pr-504.herokuapp.com/en-US/privacynotincluded/

File: public/assets/buyers-guide/menu-burger.svg [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        <g id="Group" transform="translate(944.000000, 23.000000)" fill="#000000">
+            <g id="Menu" transform="translate(2.000000, 0.000000)">
+                <g id="lines-copy">
+                    <polygon id="Rectangle-15" points="29 0 11.8460165 0.00246529159 11 2.99936385 29 3"></polygon>
+                    <polygon id="Rectangle-15-Copy-3" points="29 18 5.83961133 18.0024653 5 20.9993639 29 21"></polygon>
+                    <polygon id="Rectangle-15-Copy-2" points="29 9 0.838104339 9.00246529 1.79412041e-12 11.9993639 29 12"></polygon>
+                </g>
+            </g>
+            <text id="menu" font-family="OpenSans-Bold, Open Sans" font-size="10" font-weight="bold" letter-spacing="0.800000012">
+                <tspan x="0.46396482" y="35">menu</tspan>
```
Comment (TheoChevalier) [inline public/assets/buyers-guide/menu-burger.svg]: Is there any way this string can be added into the template instead? Maybe displaying an element with some pseudo class?

Comment (gvn) [reply public/assets/buyers-guide/menu-burger.svg]: We could do that, but it won't render as consistently as SVG. Another option would be to remove the text. @beccaklam which would you prefer?

**Comment by beccaklam** [comment]
Yes! @gvn sorry I saw it this morning but got distracted. Will get on it right away.

**Comment by beccaklam** [comment]
@gvn:
So some quick feedback:

- [ ] We should use the white mozilla logo for the menu:

moz-logo-one-color-white-rgb

- [ ] Can we center the menu content?  And add a margin-bottom of somewhere around 5-10px for each div.link-container?
screen shot 2018-01-29 at 1 12 37 pm



- [ ] This is sort of unrelated but related to the landing page -- We need more padding on the left and right sides of the page, can we change:

> .buyers-guide-header-container {
> padding: 20px 45px; to padding: 20px 60px 24px
> }

and (actually not sure if this is the right selector but see image below)

> .buyers-guide-home .categories-container .home-category-container .category-top {
> padding: 0 45px to padding: 0 60px
> }

screen shot 2018-01-29 at 1 04 09 pm

**Comment by gvn** [comment]
@beccaklam Ok, made some of those tweaks. I'm not sure what you mean by centering the menu content exactly.

Do you mean setting a max-width on the container or vertically centering it in the viewport? The comps look left aligned...

**Comment by beccaklam** [comment]
@gvn Oh sorry I just meant centering the content. Setting the div that holds menu content to 

> margin: auto 0;

 seems to do it.
screen shot 2018-01-29 at 3 12 53 pm

**Comment by beccaklam** [comment]
^ also I am still seeing a black logo @gvn am I using a wrong staging link?

**Comment by gvn** [comment]
You may have an older version cached. Try holding shift and clicking refresh.

So this is really what you want? Looks odd to me:

screen shot 2018-01-29 at 1 05 33 pm

**Comment by beccaklam** [comment]
@gvn haha it's switched around:

> margin: auto 0;

instead of

> margin: 0 auto;

**Comment by beccaklam** [comment]
@gvn also shift+refresh did not work. it's also still black in your last screen cap? i'm talking about the Mozilla logo in the full screen menu

**Comment by gvn** [comment]
Ok, the filter wasn't working properly x-browser, but should be fixed now. I also tweaked the margin.

https://advocacy-mozilla-org-st-pr-504.herokuapp.com/en-US/privacynotincluded/

**Comment by beccaklam** [comment]
@gvn Mobile view is looking good but needs some tightening up:
Current state:
screen shot 2018-01-29 at 4 44 39 pm

- [ ] The favicon should be reversed -- white text on black square (see comps)

- [ ] privacy not included should be left aligned with the asterisk sitting outside like so (not sure how you would do this in code but if you need an svg or png I can export one for you):
category page

**Comment by gvn** [comment]
Made those tweaks...

File: public/assets/buyers-guide/menu-burger.svg (hunk 1 above)
Comment (alanmoo) [reply public/assets/buyers-guide/menu-burger.svg]: Long shot, but using the l10n templating string doesn't work inside the SVG, does it?

Comment (TheoChevalier) [reply public/assets/buyers-guide/menu-burger.svg]: I would be very surprised if it does

**Comment by beccaklam** [comment]
@gvn One more menu design QA:
Can we shift the X over to where I've drawn the red X in the following image:
screen shot 2018-01-30 at 12 04 19 pm

**Comment by gvn** [comment]
Ok, tweaks made. @beccaklam can you take another look at staging?

**Comment by beccaklam** [comment]
@gvn Just looked. Soooo much better. Sorry I made a mistake:

- [ ] For the class "bg-subhead", font-size should be 12px not 23px for the mobile media query

**Comment by gvn** [comment]
K...yeah, thought that looked weird.

**Comment by beccaklam** [comment]
@gvn and 

- [ ] *privacy not included can be one line on desktop screens like before (just want mobile stacked):
screen shot 2018-01-30 at 1 44 29 pm

- For mobile screens can we 

- [ ] change the font size of *privacy not included to 16px and the line-height to 16.75px;

After that we should be good to ship! Thank you so much 💯

**Comment by gvn** [comment]
Ok, I removed the forced break between "privacy" and "not included".

I added an additional breakpoint below 400px to reduce to 12px because it was getting really crowded at 16px.

**Comment by beccaklam** [comment]
@gvn Design QA:

# **P1**

**Universal**
- [ ] We want to get rid of the Playfair typeface entirely. Can you replace all Playfair instances with Zilla Slab? 

- Menu on Product Pages (this relates to the above)

- [ ] 	change Playfair font to Open Sans

# **P2**

**Universal**
- [ ] 	change “ *privacy not included” font-weight from 900 to 400; 

**Mobile**
- All Pages
- [ ] “ *privacy not included ” should be stacked (with “not included” under “privacy”) BUT ONLY on mobile 

**Tablet/Desktop**
- [ ] Twitter icon font-size at tablet and desktop views should be at 24px to keep it consistent with the other social media icons (for some reason it’s just smaller than the other icons and that makes it look off) - P2

# **P3**

**Mobile**
- Product Page
- [ ] Can we change the related products divs to percentages so that they’re responsive? It looks strange that they’re stacked off to the left side. So somthing like this, you can play around with the numbers to see what works best:
screen shot 2018-01-30 at 2 43 08 pm

**Comment by gvn** [comment]
Ok, made those changes minus the stacking of the logo text because I think that should adapt to the width of its parent container size instead of a breakpoint if we leave it as live responsive typography.

Alternatively, we can rasterize it as SVG in two different layouts, but that's going to be a pain for localization as new assets will need to be generated for every language.

**Comment by beccaklam** [comment]
@gvn thanks changes look so good! I'm fine with leaving *privacy not included the way it is for now. Just some QA:

- [ ] The menu on product pages is still in Playfair. Again, probably some old janky styling where it's getting over ridden somewhere.
screen shot 2018-01-31 at 10 30 16 am

- [ ] Found a bug on mobile where when the pop up comes because of the sticky header the user can't close the pop up and essentially is stuck on the page. This should probably get fixed:
screen shot 2018-01-31 at 11 01 13 am

- [ ] I checked mobile for the related products and they're still sitting oddly to the left. Not a blocker to ship but would be nice to fix eventually.
screen shot 2018-01-31 at 10 57 31 am

Otherwise we're good to go 👍

**Comment by alanmoo** [comment]
Merging, Rebecca will create a new ticket with remaining QA items.

**Pull request merged**